module-gui/gui/core/DrawCommand.cpp

```diff
@@ -228,18 +228,19 @@ namespace gui
         uint32_t offsetRowContext = 0;
         uint32_t imageOffset      = 0;
         uint8_t alphaColor        = vecMap->getAlphaColor();
+        checkImageSize(ctx, vecMap);
 
         for (uint32_t row = 0; row < std::min(vecMap->getHeight(), ctx->getH()); row++) {
-            checkImageSize(ctx, vecMap);
-            uint16_t vecCount = *(vecMap->getData() + imageOffset);
+
+            uint16_t vecCount = *reinterpret_cast<const uint16_t *>(vecMap->getData() + imageOffset);
```

Comment:
would you be so kind to provide a solution that doesn't require reinterpret_cast, i.e. using bitwise operations to glue to uint8_t :)

Reply:
Oh, you want something endian neutral?  Sure... but it's not going to help since GitHub keeps claiming I have not signed the CLA.

Reply:
Oh, the CLA signing thing needed cookies enabled for their site.  Now to get rebasing on the repo working.

Reply:
I did the usual rebase and `git push --force` for the changes and it appears in my GH branch fine.  However, the changes are not showing up here as being force pushed and an older version of the branch is being referenced.  I think this might be related to my account no longer being part of the "mudita" group.

Reply:
could you please try to reset --hard your branch to the `mudita:master` and then cherry-pick/add your changes?  I don't think that's related with the group unless you're getting "rejected" message